install/roles/images/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+# Obtain/Upload guest_images to OpenStack Cloud
+
+- name: Fetch image
+  get_url:
+    url: "{{ guest_images[item].url_centos7 }}"
+    dest: "/home/stack/{{ guest_images[item].name }}.{{ guest_images[item].type }}"
+  with_items: "{{ guest_images }}"
+  when: ansible_distribution_major_version < '8'
+
+- name: Fetch image
```

Comment:
Perhaps change the name so you can tell when you are fetching images for CentOS 7 vs 8?  Or just make one role where you se the URL for "centos" to be the right one for 7 vs 8?